README.md

```diff
@@ -135,11 +228,13 @@ The following examples show numeric separators as they appear in other programmi
 
 ## Acknowledgements
 
-This strawnman proposal was developed with @ajklein, @domenic and @rwaldron.
+This proposal was developed with @ajklein, @domenic, @rwaldron and @tdd.
```

Comment:
Now that this is merged and is out of my private repo, add my name to the list too?